src/cmd/services/m3dbnode/config/pooling.go

```diff
@@ -125,6 +125,16 @@ var (
 			refillLowWaterMark:  defaultRefillLowWaterMark,
 			refillHighWaterMark: defaultRefillHighWaterMark,
 		},
+		"bufferBucket": poolPolicyDefault{
+			size:                262144,
```

Comment:
Any thoughts on what a good default here would be?

Reply:
Hm, maybe just `65536`.